docs/getting-started-guides/scratch.md

```diff
@@ -0,0 +1,664 @@
+  - Build your own image.
+  - Build your own image using `kubernetes/cluster/images/etcd/Dockerfile` and
+   `kubernetes/cluster/images/etcd/Makefile`
+
+The remainder of the document assumes that the image identifiers have been chosen and stored in corresponding env vars.  Examples (replace with latest tags and adjust if using private registry):
+  - `APISERVER_IMAGE=gcr.io/google_containers/kube-apiserver:$TAG`
+  - `SCHEDULER_IMAGE=gcr.io/google_containers/kube-scheduler:$TAG`
+  - `CNTRLMNGR_IMAGE=gcr.io/google_containers/kube-controller-manager:$TAG`
+
+### Configuring Base Node Software
+This section discusses how to configure machines to be kuberetes nodes. 
+
+You should run three daemons on every node:
+  - docker or rkt
+  - kubelet
+  - kube-proxy
+
+You will also need to do assorted other configuration on top of a
+base OS install.
+
+One possible starting point is to setup a cluster using an existing Getting
+Started Guide and then copy the init.d scripts or systemd unit files from that
+cluster, and then modify them for use on your custom cluster.
+
+#### Configuring docker
+All nodes should have docker installed.
+  - Minimum required docker or rkt version will vary as Kubernetes version changes.
+  - Newest stable is a good choice.
+  - Kubelet will log a warning and refuse to start pods if the version is too old, so try it.
+
+The way you configure docker will depend in whether you have chosen the routable-vip or overlay-network approaches for your network.
+Some docker options will want to think about:
+  - `--bridge cbr0`
+  - `--iptables=false`
+  - `--ip-masq=false`
+    - if have setup PodIPs to be routable, then you want this false, otherwise, docker will
+      rewrite the PodIP source-address to a NodeIP, to allow the Pod to talk to IPs outside the
+      cluster.
+    - if you are using an overlay network, consult those instructions.  
+  - `--bip=` and `--mtu=`
+    - may be required when using Flannel; see Flannel instructions.
+  - `--insecure-registry $CLUSTER_SUBNET` 
+    - to connect to a private registry, if you set one up, without using SSL.
+
+You may want to increase the number of open files for docker:
+   - `DOCKER_NOFILE=1000000`
+
+Ensure docker is working correctly on your system before proceeding with the rest of the
+installtion, by following examples given in the Docker documentation.
+
+Remember to eliminate any network configuration created as part of running docker previously
+with different options:
+```
+iptables -t nat -F 
+ifconfig docker0 down
+brctl delbr docker0
+```
+#### Configuring rkt
+
+*TODO*: write this
+
+#### Configuring kubelet
+
+All nodes should run kubelet.  See [Selecting Binaries](#selecting-binaries).
+
+Arguments to consider:
+  - If following the HTTPS security approach:
+    - `--api_servers=https://$MASTER_IP`
+  - Otherwise, if taking the firewall-based security approach
+    - `--api_servers=http://$MASTER_IP`
+    - `--kubeconfig=/var/lib/kube-proxy/kubeconfig`
```

Comment:
Fixed.
You are right that the flag is not set in the default file.  The kubelet is using the default value of the flag.  I thought it was clearer to set it explicitly in this guide, so that the user understands the file needs to be present and so user can change path to file.